src/test/git-elegant-configure-repository.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+load addons-common
+load addons-read
+load addons-fake
+load addons-git
+
+fake-preconditions() {
+    fake-pass git "elegant commands"
+
+    fake-pass git "config user.name" "UserName"
+    fake-pass git "config user.email" "UserEmail"
+
+    fake-pass git "config --local core.commentChar"
+
+
+    fake-pass git "config --local user.name UserName"
+    fake-pass git "config --local user.email UserEmail"
+
+    fake-pass git "config --local apply.whitespace fix"
+}
+
+teardown() {
+    clean-fake
+    clean-git
+}
+
+@test "'configure': '--local' option is available" {
```

Comment:
@alexbeatnik this option is unavailable. WHy do we have this test?

Reply:
@extsoft Thanks for review, fixed